moto/eks/models.py

```diff
@@ -631,6 +631,57 @@ def delete_nodegroup(self, cluster_name, nodegroup_name):
         cluster.nodegroup_count -= 1
         return result
 
+    def tag_resource(self, resource_arn, tags):
+        try:
```

Comment:
Can we have a docstring here to say that we only support tagging Clusters, and not managed node groups?

Reply:
yes, will update, thanks